src/types/toaster.ts

```diff
@@ -2,14 +2,14 @@ export type ToasterAppearance = 'info' | 'success' | 'danger' | 'warning' | 'sys
 
 export interface Toast {
   /** Unique identifier of toaster */
-  key?: any
+  key: string
   /** Title of toaster */
   title?: string
   /** Text to display in toaster */
   message?: string
   appearance?: ToasterAppearance
   timeoutMilliseconds?: number
-  timer?: any
+  timer?: number
```

Comment:
question: the interface is changing here; is that going to create a downstream issue?